### DISCUSSION TITLE
Plotting (IN RAM)

### DISCUSSION THREAD

**Discussion by Proulx417** [Chia-Network/chia-blockchain#3617] [Q&A]
I'm buying more ram for my server like 768gb for plotting. Is it a bad idea? If not what is the best command to plot without HDD/SSD?

**Comment by justinphelps** [comment]
More RAM never hurts system performance. Assuming you're running linux, check out [tmpfs](https://www.kernel.org/doc/html/latest/filesystems/tmpfs.html)
Don't expect the 10X performance gains though. 
Some people have done it with about [~40%](https://chiaforum.com/t/ramdrive-vs-ssd/1563/5) speed improvement over SSD , but only one at a time. In aggregate, you can get much more TB/d out of lots of plots in parallel on SSDs. 
The long term benefit is of course that you don't have to throw away your SSDs.
